spec/features/admin/order_spec.rb

```diff
@@ -144,7 +144,7 @@ def new_order_with_distribution(distributor, order_cycle)
       fill_in(:quantity, with: max_quantity + 1)
       find("a.save-item").click
     end
-    accept_js_alert
+    find("button.confirm").click
```

Comment:
Shouldn't we test for what the user sees? What do you think about this?
```suggestion
    click_button("OK")
```